src/tools/create-lambda-handler.js

```diff
@@ -130,14 +130,8 @@ const loadApp = (event, rpc, appRawOrPath) => {
   });
 };
 
-const createLambdaHandler = (appRawOrPath, { skipHttpPatch } = {}) => {
+const createLambdaHandler = appRawOrPath => {
```

Comment:
I was going to say this looked dangerous because I thought any existing calls to `createLambdaHandler` that passed more than one argument would break, but it turns out JavaScript is pretty "meh" on enforcing function arity and would just drop it any way. 

It still might be good to fix any existing calls to this that have more than one argument provided (the one in `src/tools/create-app-tester.js` was the only one that I saw via a GitHub search but I didn't look very hard) in order to reduce future confusion (or errors if a new parameter is introduced later).

Reply:
yeah, js don't care. Typescript does though. In any case, good catch!